### PR TITLE
Update rules_xcodeproj to 1.3.2

### PR DESCRIPTION
**Pull request by mattrobmattrob**

### CODE REVIEW

**Review by brentleyjones — APPROVED** [review]

**Comment by brentleyjones** [comment]
~You can remove the `.rules_xcodeproj` .gitignore~ (after you adjust the visibility statements, if needed).

File: modules/BUILD.bazel [hunk 1]
```diff
@@ -5,8 +5,6 @@ package_group(
         "//app",
         # All other modules.
         "//modules/...",
-        # Project generation.
-        "//.rules_xcodeproj/...",
```
Comment (brentleyjones) [inline modules/BUILD.bazel:9]: You'll need `@rules_xcodeproj_generated//` now

Comment (mattrobmattrob) [reply modules/BUILD.bazel:9]: Was curious about that.

File: modules/BUILD.bazel [hunk 2]
```diff
@@ -6,7 +6,7 @@ package_group(
         # All other modules.
         "//modules/...",
         # Project generation.
-        "@rules_xcodeproj_generated//...",
+        "@rules_xcodeproj//:generated",
```
Comment (brentleyjones) [inline modules/BUILD.bazel]: Needs to be in `includes`

Comment (mattrobmattrob) [reply modules/BUILD.bazel]: This is what I get for trying to just use Codespaces to do it all without ever having it locally.

File: modules/BUILD.bazel [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 
 package_group(
     name = "default_test_visibility",
-    packages = [
-        # The root package to be included in the Xcode project.
-        "//",
+    includes = [
         # Project generation.
-        "//.rules_xcodeproj/...",
+        "@rules_xcodeproj//:generated",
```
Comment (brentleyjones) [inline modules/BUILD.bazel]: I had a typo: "@rules_xcodeproj//xcodeproj:generated"

**Pull request merged**